java/android/nnstreamer/src/main/jni/Android-llama2c-prebuilt.mk

```diff
@@ -0,0 +1,30 @@
+#------------------------------------------------------
+# Llama2.c: Inference Llama 2 in one file of pure C
+#   https://github.com/karpathy/llama2.c (original upstream repository)
+#   https://github.com/nnsuite/llama2.c (NNSuite forked repository)
+#
+# This mk file defines a prebuilt shared library for the llama2.c module.
+#------------------------------------------------------
+LOCAL_PATH := $(call my-dir)
+
+LLAMA2C_LIB_PATH := $(LLAMA2C_DIR)/lib
+
+ifndef LLAMA2C_LIB_PATH
```

Comment:
It would be LLAMA2C_DIR, not LLAMA2C_LIB_PATH.
Also, does llama2c compiled when building nnstreamer.aar? Then I suggest adding 'llama2c' module in 'Android-llama2c.mk' like tensorflow-lite.

Reply:
> Does llama2c compiled when building nnstreamer.aar?
Yes, it does.

> 
- [java/android/nnstreamer/src/main/jni/Android-llama2c.mk](https://github.com/nnstreamer/api/pull/563/files#diff-3c2e508aec90de520153b44c97ec1bdd08025be0bdecb1c5c8fd74eb32f104c8) is to build libs
- [java/android/nnstreamer/src/main/jni/Android-llama2c-prebuilt.mk](https://github.com/nnstreamer/api/pull/563/files#diff-1284da35bf9562d3b0c171bf5ddb402b02dd21fd34fe6f43f330a6fde2583c0f), which is imported by [java/android/nnstreamer/src/main/jni/Android-llama2c.mk](https://github.com/nnstreamer/api/pull/563/files#diff-3c2e508aec90de520153b44c97ec1bdd08025be0bdecb1c5c8fd74eb32f104c8), is to use prebuilt libraries whether there are prebuilt libs or not

Reply:
Replacing ```LLAMA2C_LIB_PATH``` with ```LLAMA2C_DIR``` has been applied.